modules/oe_content_event/config/install/core.entity_form_display.node.oe_event.default.yml

```diff
@@ -160,37 +161,27 @@ content:
     region: content
   created:
     type: datetime_timestamp
-    weight: 25
+    weight: 26
     region: content
     settings: {  }
     third_party_settings: {  }
-  oe_event_programme:
-    weight: 39
+  field_oe_event_status_descriptio:
```

Comment:
This needs to be fixed to oe_event_status_description.

Reply:
It actually needs to be removed cause it's duplicated. The proper one is lower.